packages/xo-web/src/xo-app/dashboard/overview/index.js

```diff
@@ -68,29 +70,72 @@ class PatchesCard extends Component {
   }
 }
 
-@connectStore(() => {
-  const getHosts = createGetObjectsOfType('host')
-  const getVms = createGetObjectsOfType('VM')
+@connectStore({
+  hosts: createGetObjectsOfType('host'),
+  pools: createGetObjectsOfType('pool'),
+  srs: createGetObjectsOfType('SR').filter([isSrWritable]),
+  vms: createGetObjectsOfType('VM'),
+  alarmMessages: createGetObjectsOfType('message').filter([
+    message => message.name === 'ALARM',
+  ]),
+  tasks: createGetObjectsOfType('task').filter([
+    task => task.status === 'pending',
+  ]),
+})
+@addSubscriptions({
+  plugins: subscribePlugins,
+  users: subscribeUsers,
+})
+@injectIntl
+class DefaultCard extends Component {
+  _getPoolWisePredicate = createSelector(
+    () => map(this.state.pools, 'id'),
+    poolsIds => item => isEmpty(poolsIds) || includes(poolsIds, item.$pool)
+  )
 
-  const getHostMetrics = createGetHostMetrics(getHosts)
+  _getPredicate = createSelector(
+    this._getPoolWisePredicate,
+    () => map(this.state.hosts, 'id'),
+    (poolWisePredicate, hostsIds) => item =>
+      poolWisePredicate(item) &&
+      (isEmpty(hostsIds) ||
+        includes(hostsIds, item.$container || item.$host))
```

Comment:
```js
isEmpty(hostIds)
  ? poolWisePredicate(item)
  : includes(hostsIds, item.$container || item.$host)
```
since if you're in the host, you're necessarily in the pool.